DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -218,24 +220,24 @@ In this tutorial, we will show how to use the `kernel_args_restrict` attribute f
+Compare the loop initiation interval (II) between the kernels with or without `[[intel::kernel_args_restrict]]` attribute. Notice that the loop in the *KernelArgsNoRestrict_Lambda* and *KernelArgsNoRestrict_Functor* kernels has a large estimated II, while the loop in the *KernelArgsRestrict_Lambda* and *KernelArgsRestrict_Functor* kernels has an estimated II of ~1. These IIs are estimates because the latency of global memory accesses varies with runtime conditions.
 
-For the *KernelArgsNoRestrict* kernel, the compiler assumed that the kernel arguments can alias each other. Since`out[i]` and `in[i+1]` could be the same memory location, the compiler cannot overlap the iteration of the loop performing `out[i] = in[i]` with the next iteration of the loop performing `out[i+1] = in[i+1]` (and likewise for iterations `in[i+2]`, `in[i+3]`, ...). This results in an II equal to the latency of the global memory read of `in[i]` plus the latency of the global memory write to `out[i]`.
+For the *KernelArgsNoRestrict_Lambda* and *KernelArgsNoRestrict_Functor* kernel, the compiler assumed that the kernel arguments can alias each other. Since`out[i]` and `in[i+1]` could be the same memory location, the compiler cannot overlap the iteration of the loop performing `out[i] = in[i]` with the next iteration of the loop performing `out[i+1] = in[i+1]` (and likewise for iterations `in[i+2]`, `in[i+3]`, ...). This results in an II equal to the latency of the global memory read of `in[i]` plus the latency of the global memory write to `out[i]`.
 
-We can confirm this by looking at the details of the loop. Click the *KernelArgsNoRestrict* kernel in the *Loop List* pane and then click the loop in the *Loop Analysis* pane. Now consider the *Details* pane below. You should see something like:
+We can confirm this by looking at the details of the loop. Click the *KernelArgsNoRestrict_Lambda* kernel (similarly for the *KernelArgsNoRestrict_Functor* kernel) in the *Loop List* pane and then click the loop in the *Loop Analysis* pane. Now consider the *Details* pane below. You should see something like:
 
-- *Compiler failed to schedule this loop with smaller II due to memory dependency*
-  - *From: Load Operation (kernel_args_restrict.cpp: 74 > accessor.hpp: 945)*
-  - *To: Store Operation (kernel_args_restrict.cpp: 74)*
+- *Compiler failed to schedule this loop with smaller II due to memory dependency:*
+  - *From: Load Operation (handler.hpp: 1502 > kernel_args_restrict.cpp: 94)*
+  - *To: Store Operation (handler.hpp: 1502 > kernel_args_restrict.cpp: 94)*
 - *Most critical loop feedback path during scheduling:*
-  - *144.00 clock cycles Load Operation (kernel_args_restrict.cpp: 74 > accessor.hpp: 945)*
-  - *42.00 clock cycles Store Operation (kernel_args_restrict.cpp: 74)*
+  - *32.00 clock cycles Load Operation (handler.hpp: 1502 > kernel_args_restrict.cpp: 94)*
+  - *31.00 clock cycles Store Operation (handler.hpp: 1502 > kernel_args_restrict.cpp: 94)*
```

Comment:
Does this text match what the report says? The text in the report may have changed since this code sample was last updated.